External/FEXCore/Source/Interface/HLE/Syscalls/FD.cpp

```diff
@@ -293,6 +293,12 @@ namespace FEXCore::HLE {
     SYSCALL_ERRNO();
   }
 
+  uint64_t Pselect6(FEXCore::Core::InternalThreadState *Thread, int nfds, fd_set *readfds, fd_set *writefds, fd_set *exceptfds, const struct timespec *timeout, const sigset_t *sigmask) {
+    // sig is ignored here? -> need to validate w/ glibc source
```

Comment:
What sig here?

Reply:
oh, the comment is outdated, will remove it.